### ISSUE TITLE
Inconsistent Migration History after upgrade to NB2.0

### ISSUE THREAD

**Issue by JimmyKip** [nautobot/nautobot-plugin-device-lifecycle-mgmt#234]
### Environment
* Python version:  3.11
* Nautobot version:  2.0.1
* nautobot-plugin-device-lifecycle-mgmt version:  2.0.0

### Expected Behavior
Nautobot 2.0.1 to start successfully after adding Device Lifecycle Management Plugin to image.

### Observed Behavior
Received Inconsistent Migration Error as Nautobot launched:
```
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration dcim.0031_remove_device_role_and_rack_role is applied before its dependency nautobot_device_lifecycle_mgmt.0016_role_migration_cleanup on database 'default'.
```

### Steps to Reproduce
1. Install Device Lifecycle Management 1.3.2  in < 2.0 Nautobot
2. Upgrade Nautobot to 2.0.1 with no plugins installed or enabled
3. Install & enable Device Lifecycle Management 2.0.0 in Nautobot 2.0.1
4. Receive DB error on launch, NB goes into crash loop waiting for DB.

**Comment by glennmatthews** [comment]
Related: nautobot/nautobot#4609. Broadly speaking the workflow for upgrading to 2.0 looks likely to *require* that the apps you had enabled in 1.x remain enabled during the upgrade, rather than enabling them after the upgrade is completed, but we're looking into whether we can make improvements in this regard.

**Comment by JimmyKip** [comment]
Cheers Glenn, that makes sense.

**Comment by glennmatthews** [comment]
Fixed by #236, needs a new release though.
